src/solidity-frontend/solidity_convert.cpp

```diff
@@ -4861,14 +4909,16 @@ void solidity_convertert::get_function_definition_name(
   //  - For function name, just use the ast_node["name"]
   // assume Solidity AST json object has "name" field, otherwise throws an exception in nlohmann::json
   std::string contract_name;
-  assert(ast_node.contains("scope"));
```

Comment:
This `"scope"` hasn't been needed?

Reply:
Yes, it was used for debugging, but it is no longer needed.